src/components/MessagesList/MessagesGroup/Message/MessagePart/AudioPlayer.vue

```diff
@@ -72,8 +83,44 @@ export default {
 		},
 	},
 
+	mounted() {
+		EventBus.on('audio-player-ended', this.autoPlay)
+	},
+
+	beforeDestroy() {
+		EventBus.off('audio-player-ended', this.autoPlay)
+	},
+
 	methods: {
 		t,
+
+		handleEnded() {
```

Comment:
Would be better to move all the  computation logic here:
- handleEnded will be executed once
- autoPlay - for each component on page

Reply:
Yes makes sense! Done!